eng/Version.Details.xml

```diff
@@ -293,6 +293,14 @@
       <Uri>https://github.com/dotnet/corefx</Uri>
       <Sha>0abec4390b30fdda97dc496594f9b1f9c9b20e17</Sha>
     </Dependency>
+    <Dependency Name="System.ComponentModel" Version="4.3.0">
+      <Uri>https://github.com/dotnet/corefx</Uri>
+      <Sha>000000</Sha>
+    </Dependency>
+    <Dependency Name="System.ComponentModel.Annotations" Version="4.6.0-preview.19109.6">
+      <Uri>https://github.com/dotnet/corefx</Uri>
+      <Sha>000000</Sha>
```

Comment:
```suggestion
      <Sha>0abec4390b30fdda97dc496594f9b1f9c9b20e17</Sha>
```

Reply:
Since we know the sha, let's use it. 000000 was meant to be a suggestion if you _don't_ know the right sha to use.

Reply:
Thanks - done.